adabot/lib/circuitpython_library_validators.py

```diff
@@ -1210,3 +1211,13 @@ def validate_passes_linting(self, repo):
                     f.write(''.join(pylint_result))
 
         return []
+
+    def validate_default_branch(self, repo):
+        """ Makes sure that the default branch is main """
+        if not repo["name"].startswith("Adafruit_CircuitPython"):
+            return []
+
+        if repo["default_branch"] != "main":
+            return[ERROR_INCORRECT_DEFAULT_BRANCH]
```

Comment:
is this repo pylinted ? I'm surprised it's OK without a space here.

Reply:
It isn't. Seems like there's about 400-500 things that need to be fixed.